longest_palindromic_substring/longest_palindromic_substring.cpp

```diff
@@ -0,0 +1,61 @@
+//  Implementation of dp problem: Longest Palindromic substring
```

Comment:
This implementation also looks logically incorrect.
Check your code here : http://ideone.com/A6fpRi